src/main/java/core/Parser.java

```diff
@@ -0,0 +1,61 @@
+package core;
+
+import commands.AddCommand;
```

Comment:
I like how imports are explicit and not replaced with wildcard imports, making the functionality of the code much more readable and understandable.

Reply:
Troublesome but had to follow the standards